code/modules/mob/mob_grab.dm

```diff
@@ -37,7 +37,7 @@
 
 
 //Used by throw code to hand over the mob, instead of throwing the grab. The grab is then deleted by the throw code.
-/obj/item/weapon/grab/proc/throw()
+/obj/item/weapon/grab/proc/throw_target()

```

Comment:
Oooh. Yeah, it is, if you're using the updated BYOND, I forgot about it completely.